components/brave_rewards/resources/request_ads_enabled/lib/host.ts

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Host, HostState } from './interfaces'
+import { createStateManager } from '../../shared/lib/state_manager'
+
+const rewardsTourURL = 'brave_rewards_panel.html#tour'
+
+export function createHost (): Host {
+  const port = chrome.runtime.connect({ name: 'request-enable-rewards-panel' })
+
+  const stateManager = createStateManager<HostState>({
+    loading: true,
+    rewardsEnabled: false
```

Comment:
we need to use consistent terminology and either pick ads or rewards, but not use them interchangeably cc @tmancey

Reply:
This parameter is used to identify, should we suggest `Enable Rewards` popup instead of `Enable Ads` popup.
I've renamed this as `showRewardsOnboarding` to remove misunderstanding.